CMakeLists.txt

```diff
@@ -235,6 +235,8 @@ add_subdirectory(components)
 add_subdirectory(data)
 add_subdirectory(services)
 add_subdirectory(src)
+
+enable_testing()
```

Comment:
Alternately, one might `include(CTest)` and rely on the *BUILD_TESTING* option that that introduces. But that's nit-picking: this is a good change because otherwise you have tests that cannot be run.

Reply:
That's exactly why I added it.